### ISSUE TITLE
Home Page: Add "Edit" link to contextual menu for editing home page hero block

### ISSUE THREAD

**Issue by mshaver** [thinkshout/bene#108]
## Overview
- [ ] Add "Edit" link to home page hero block
![fullscreen_9_12_17__4_31_pm](https://user-images.githubusercontent.com/179803/30352990-2d07aab8-97d8-11e7-8d66-7348fa3ee695.png)

**Comment by tauno** [comment]
This requires a bit more thought. There are two pieces of functionality here:

1. Configure which home page hero block is displayed (this is the current Configure block functionality)
2. Edit the contents of the particular home page hero block that is currently displayed on the home page.

For MVP we likely need to at least add help text and an edit link to the configure block page.

Other thoughts @mshaver or @juliaford about how to make this a better experience?

**Comment by juliaford** [comment]
I think you could just get rid of the pencils entirely and they can navigate to the content other ways (edit slider on bottom of page or via main nav menu). Either that or only have the pencil allow for content editing. @mshaver thoughts?

**Comment by tauno** [comment]
I got the edit link for the block itself to show in addition to the configure block one. Downside is there are now two contextual links where the difference is a bit confusing. "Configure" allows you to change the block being shown and "Edit" allows you to edit the actual content of the block:
![image](https://user-images.githubusercontent.com/250188/30718135-d98c906c-9ed3-11e7-896e-d5b6bbc5e30e.png)

I also added a Blocks page under content where they can manage the content.
